shuttle/db/db_functions.py

```diff
@@ -102,3 +104,56 @@ def username_search(username):
             conn.close()
         # if mybethel can't get the data, then prevent anything from loading
         return abort(503)
+
+
+def commit_schedule_to_db(table, all_locations):
+    sql = "DELETE FROM SHUTTLE_SCHEDULE"
+    query(sql, 'write')
+    all_locations = [i.upper() for i in all_locations]
+    for i in range(len(table)):
+        location = ""
+        for j in range(len(table[i])):
+            arrival_time = 0
+            if table[i][j].upper() in all_locations:
+                location = table[i][j].upper()
+            elif table[i][j] == '-' or table[i][j] == 'DROP':
+                arrival_time = '01-AUG-00 01.00.00.000000000 AM'
```

Comment:
So if my understanding is right you are still submitting a row when it isn't a valid time (- or DROP). Instead of doing that just skip the row.